examples/door-lock-app/linux/include/LockEndpoint.h

```diff
@@ -42,8 +42,8 @@ class LockEndpoint
 
     inline chip::EndpointId GetEndpointId() { return mEndpointId; }
 
-    bool Lock(const Optional<chip::ByteSpan> & pin);
-    bool Unlock(const Optional<chip::ByteSpan> & pin);
+    bool Lock(const Optional<chip::ByteSpan> & pin, DlOperationError & err);
```

Comment:
`DlOperationError` doesn't have a _success_ code defined, just set of error codes. It will be impossible to return OK from the functions in such a case.